content/06.know-your-problem.md

```diff
@@ -1,41 +1,40 @@
-For example, are some samples collected from the same individuals at different time points?
-Are those time points before and after some treatment?
-If one assumes that all the samples are independent but that is in fact not the case, a variety of issues may arise, including having a lower effective sample size than expected.
-As described in [Tip 1](#concepts), unsupervised learning and other exploratory analyses can be identify such biases in these datasets prior to applying the deep learning model.
+For example, the existence in a dataset of samples collected from the same individuals at different time points can have signficant effects on analyses that make assumptions about sample size and independence (_i.e._ non-independence can lower the effective sample size).
+Another potential issue is the existence of systematic biases, which can be induced by confounding variables and can lead to artifacts or so-called "batch effects."
+As a consequence, models may learn to rely on the correlations that these systematic biases underpin, even though they are irrelevant to the scientific context of the study.
+This can lead to misguided predictions and misleading conclusions [@doi:10.1038/nrg2825].
+As described in [Tip 1](#concepts), unsupervised learning and other exploratory analyses can help to identify such biases in these datasets prior to applying a deep learning model.
 
-In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Tip 7](#overfitting)).
-For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-It is important to note that molecular or imaging datasets often require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
-Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+In general, deep learning has an increased tendency towards overfitting (i.e. perform well on the training data but not generalize well) compared to classical methods. This is often a result of the large number of parameters being estimated in a DL model, and has direct implications for the importance of working with adequate sample sizes (see [Tip 7](#overfitting)).
+For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations, and thus incorrect interpretations or conclusions (see [Tip 9](#interpretation)).

```

Comment:
Should we delete these lines? There doesn't seem to be a strong reason for them to be here and not in the overfitting tip

Reply:
The only reason I can think of to potentially keep them is the value of context and repetition for the less familiar reader. In that sense, it may increase accessibility and be a bit more "digestible". Otherwise, I agree with @Benjamin-Lee .

Reply:
There are quite a few tips with these cross references. As I'm reading I do find them a bit odd for this type for a manuscript that is not _that_ long. I'd probably remove them throughout, unless someone has a strong case for keeping them.

Reply:
I don't have a strong case to keep them. The only reason I can think to hold onto them is to make sure that the reader knows that the topic being discussed will be more fully discussed elsewhere. @cgreene is right that it is a bit strange to have links in a paper of this length. I think having fig 1 be a list of the tips will ensure that the reader knows what's going to come. Raised in #266